configure.ac

```diff
@@ -861,10 +868,88 @@ AM_CONDITIONAL([OPENTHREAD_EXAMPLES_SAMR21],    [test "${with_examples}" = "samr
 
 AM_COND_IF([OPENTHREAD_EXAMPLES_POSIX], CPPFLAGS="${CPPFLAGS} -DOPENTHREAD_EXAMPLES_POSIX=1", CPPFLAGS="${CPPFLAGS} -DOPENTHREAD_EXAMPLES_POSIX=0")
 
-AC_MSG_CHECKING([whether to enable examples])
-AC_MSG_RESULT(${with_examples})
-
 AM_CONDITIONAL([OPENTHREAD_EXAMPLES_NRF528XX], [test OPENTHREAD_EXAMPLES_NRF52811 || test OPENTHREAD_EXAMPLES_NRF52833 || test OPENTHREAD_EXAMPLES_NRF52840])
```

Comment:
It seems that this line does not work as one would expect. In `if` statements in `Makefile.am` the `OPENTHREAD_EXAMPLES_NRF528XX` always evaluates to true.

Reply:
This line should read as follows:

```
AM_CONDITIONAL([OPENTHREAD_EXAMPLES_NRF528XX], [test "${with_examples}" = "nrf52811" -o "${with_examples}" = "nrf52833" -o "${with_examples}" = "nrf52840"])
```